site/source/docs/porting/multimedia_and_graphics/OpenGL-support.rst

```diff
@@ -40,6 +42,8 @@ This allows you to use functions `glDrawArrays <https://www.opengl.org/sdk/docs/
 
 To enable *OpenGL ES 2.0 emulation*, specify the :ref:`emcc <emcc-s-option-value>` option ``-s FULL_ES2=1`` when linking the final executable (.js/.html) of the project.
 
+To enable *OpenGL ES 3.0 emulation*, specify the :ref:`emcc <emcc-s-option-value>` option ``-s FULL_ES3=1`` when linking the final executable (.js/.html) of the project. This adds emulation for mapping memory blocks to client side memory.
```

Comment:
is this in addition to `FULL_ES2`? That is, do people need to specify both?

lgtm with this clarified